WWTExplorer3d/WWTExplorer.csproj

```diff
@@ -1,2552 +1,2552 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think this file is churning because the line endings are changing between DOS and Unix? It would be nice to rewrite the Git history to avoid the churn. With the right magic in a toplevel `.gitattributes` file we should be able to get Git to avoid this kind of churn — although I'm not sure which exact magic we need. Git can do a normalization pass when importing/exporting a file between the working directory and what the repo actually stores, and we need to set that up so that the repo version of the file doesn't churn regardless of whether it's edited on Windows or Unix.

Reply:
Yeah, I think you're right about the line endings being the issue here, and `.gitattributes` being the solution. It seems like `* text=auto` will do what we want, possibly with some special cases like [here](https://rehansaeed.com/gitattributes-best-practices/#line-endings).

Do you know if there's a convenient way to rewrite the git history to remove the changes to this file?